public/js/lengthdistribution.js

```diff
@@ -88,6 +110,17 @@ export class Graph {
     });
   }
 
+  setupResponsiveness() {
```

Comment:
Would be great if you can set this up as a jquery extension that adds _one_ event handler to `window.resize`. The handler looks for all `.grapher` divs, fetches the corresponding ReactComponent instance and calls `.graph.draw` on that. In a second step we can optimise redrawing of the graphs by for example, debouncing them at different intervals.